packages/code-du-travail-ui/src/__tests__/BreadCrumbs.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render } from "react-testing-library";
+import BreadCrumbs from "../BreadCrumbs";
+
+describe("<Alert />", () => {
+  test("should render with default props", () => {
+    const { container } = render(<BreadCrumbs />);
+    expect(container).toMatchSnapshot();
+  });
+  test("should render width custom props", () => {
+    /* eslint-disable-next-line react/jsx-key */
```

Comment:
why ?
Can use a `//` comment if it's just this line

Reply:
since there is JSX inside an array, it trigger the eslint rule `react/jsx-key`  
ok to change the comment